2024-02-17 11:57:50.730 [info] (Client) Pylance async client (2024.2.2) started with python extension (2024.0.1)
2024-02-17 11:57:53.000 [info] [Info  - 11:57:52 AM] (48679) Pylance language server 2024.2.2 (pyright version 1.1.348, commit cfb1de0c) starting
2024-02-17 11:57:53.008 [info] [Info  - 11:57:53 AM] (48679) Server root directory: file:///root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-02-17 11:57:53.010 [info] [Info  - 11:57:53 AM] (48679) Starting service instance "root"
2024-02-17 11:57:53.157 [info] [Info  - 11:57:53 AM] (48679) Setting pythonPath for service "root": "/usr/local/bin/python3"
2024-02-17 11:57:53.166 [info] [Info  - 11:57:53 AM] (48679) Setting environmentName for service "root": "3.12.1 (グローバル)"
2024-02-17 11:57:53.248 [info] [Info  - 11:57:53 AM] (48679) Assuming Python version 3.11
2024-02-17 11:57:54.084 [info] [Info  - 11:57:54 AM] (48679) Found 43 source files
2024-02-17 11:57:54.092 [info] [Info  - 11:57:54 AM] (48679) Background analysis(1) root directory: file:///root/.vscode-server/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-02-17 11:57:54.093 [info] [Info  - 11:57:54 AM] (48679) Background analysis(1) started
